gradle.properties

```diff
@@ -1,4 +1,4 @@
-VERSION_NAME=2.1.2
+VERSION_NAME=2.1.3
```

Comment:
Nit: I think this could arguably be a minor version bump (i.e. `2.2.0`), not a patch version bump. The definitions (from http://semver.org/):

```
MINOR version when you add functionality in a backwards-compatible manner, and
PATCH version when you make backwards-compatible bug fixes.
...
A bug fix is defined as an internal change that fixes incorrect behavior.
```

I guess it's fuzzy whether this is a "bug fix" or "added functionality". I'd say the latter but if you think it's the former that's reasonable, just wanted to call out the distinction explicitly.